roles/build-grsec-kernel/tasks/compile.yml

```diff
@@ -15,15 +15,27 @@
     # to PATH, which will override the system gcc and and g++ binaries.
     PATH: "{% if grsecurity_build_use_ccache == true %}/usr/lib/ccache:{% endif %}{{ ansible_env.PATH }}"
 
+- name: Find newly build kernel packages.
+  find:
+    paths:
+      - "{{ grsecurity_build_download_directory }}"
+    patterns:
+      # Intentionally fuzzy fileglob to support fetching back multiple
+      # build targets, e.g. image, headers, src, manual, etc.
+      - linux-*-{{ linux_kernel_version }}-grsec*.deb
+  register: grsecurity_build_find_packages_result
+  tags: fetch
+
+- debug: var=grsecurity_build_find_packages_result
```

Comment:
Not usually a fan of leaving in debug statements because it can muddle up the output on (especially on register values) buttt it seems like a fair call to do here. So 👍

Reply:
The `debug` inclusion was left over from debugging, but I _am_ going to leave it in for now, simply to make the role output easier to troubleshoot in the future (since the role takes so long to run, it's difficult to try again with an ad-hoc debug task added in).